tox.ini

```diff
@@ -34,6 +34,7 @@ deps =
     pytest-verbose-parametrize==1.7.0
     pytest-xdist==1.29.0
     shade==1.31.0
+    pre-commit
```

Comment:
You need to add a upper and lower bound. Also take into account that https://github.com/ansible/molecule/pull/2163 is happening, which you might need to coordinate with.

Reply:
Understood, I'll move it next to flake8 since pre-commit is more closely related to that dep.